R/derive_params_exposure.R

```diff
@@ -166,6 +166,7 @@ derive_params_exposure <- function(dataset,
     ) %>%
     filter(PARAMCD == quo_get_expr(set_values_to$PARAMCD))
```

Comment:
`get_summary_records()` should be called here (lines 161-167).

Reply:
Thanks @bundfussr, I tried adding that, however when testing I now get an error `Error in is_quosure(arg) : object 'PARAMCD' not found`.  I will continue to look into it, in the meantime I have created a draft PR here - https://github.com/pharmaverse/admiral/pull/1118
Thanks for your help, 
Alana

Reply:
Please try adding
```
filter <- assert_filter_cond(enquo(filter), optional = TRUE)
```
in `get_summary_records()`. This quotes the `filter` argument and ensures that it is a condition.

Reply:
Thanks @bundfussr, I did try adding that, as well as wrapping the filter condition in `rlang::quo()` in `derive_param_exposure` - however I'm now getting a different error when the filtering is applied... I am probably missing something somewhere, will keep looking into it.

Reply:
In the call of `get_summary_records()` you should not use `quo()`, i.e., it should be `filter = PARAMCD == input_code`.

Reply:
We plan to add more guidance regarding quosures in the programming strategy.